patches/net/minecraft/world/flag/FeatureFlags.java.patch

```diff
@@ -0,0 +1,18 @@
+--- a/net/minecraft/world/flag/FeatureFlags.java
++++ b/net/minecraft/world/flag/FeatureFlags.java
+@@ -13,6 +_,7 @@
+     public static final Codec<FeatureFlagSet> CODEC;
+     public static final FeatureFlagSet VANILLA_SET;
+     public static final FeatureFlagSet DEFAULT_FLAGS;
++    public static final FeatureFlag MOD_EXPERIMENTAL;
```

Comment:
This needs javadoc on what the flag is for, and how modders should actually use it -- notably, they have to register their own pack using it in `AddPackFindersEvent`, which isn't necessarily obvious!

Reply:
Java docs have been added stating what this flag is, how and when to use it, and stating that it is recommended that modders document their experimental features somewhere.